### PR TITLE
Fixed Android 8.0+ service crash in background

### PR DESCRIPTION
**Pull request by marf**
#### What's this PR do?
Fix Android 8.0+ crashes
#### Which issue(s) is it related to?
#191 
#### How to test:
Now music controls in background works and do not crash

### CODE REVIEW

File: android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java [hunk 1]
```diff
@@ -156,11 +175,24 @@ private int toKeyCode(long action) {
...
             return null;
         }
 
+        @Override
+        public void onCreate() {
+            super.onCreate();
+
+            if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                /*Notification notification = new Notification.Builder(MusicControlModule.INSTANCE.context, MusicControlModule.INSTANCE.CHANNEL_ID)
+                        .build();*/
```
Comment (Guichaguri) [inline android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java]: Thank you! Can you cleanup this unused code?

**Comment by marf** [comment]
I have cleaned up the unused line, I think you can merge.

File: android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java (hunk 1 above)
Comment (marf) [reply android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java]: I have cleaned up the unused line, I think you can merge.

**Comment by marf** [comment]
I have cleaned up the unused line, I think you can merge.

**Comment by tanguyantoine** [comment]
Sorry for the delay. I published the version 0.9.0 any chance you try this one?

**Comment by marf** [comment]
I will try it, but I do not think that this will cause any problem. I have read that in the new version you just supported typescript so the Android changes in the pull should not affect it.

**Comment by tanguyantoine** [comment]
keep me in touch 🙏

**Pull request merged**